package.json

```diff
@@ -32,5 +31,9 @@
     "semver": "^7.3.2",
     "source-map-support": "^0.5.19",
     "typescript": "^3.9.5"
+  },
+  "dependencies": {
+    "axios": "^0.19.2",
+    "fetch": "^1.1.0"
```

Comment:
Don't modify the `package.json`, if you choose to make HTTP requests use the fetch function.